1.Exploratory_Data_Analysis/nbconverted/Exploratory_Data_Analysis.py

```diff
@@ -20,26 +20,29 @@
 # In[1]:
 
 
-import itertools
+import pathlib
 import sys
 
 import matplotlib.pyplot as plt
 import pandas as pd
+import pyarrow.parquet as pq
 import seaborn as sns
+import umap
 
 get_ipython().run_line_magic("matplotlib", "inline")
+import itertools
 
 from sklearn.cluster import KMeans
 from sklearn.manifold import TSNE
 
 sys.path.append("..")
-from ..utils.utils import *
+from utils.utils import df_stats, plot_features_all_cluster
```

Comment:
Make sure you follow import order: 
1 - standard lib
2 - 3rd party lib 
3 - file based imports (user made imports)
[source](https://peps.python.org/pep-0008/#imports)